src/ansys/pyensight/core/utils/omniverse_dsg_server.py

```diff
@@ -372,7 +366,7 @@ def create_dsg_lines(
     ):
         # 1D texture map for variables https://graphics.pixar.com/usd/release/tut_simple_shading.html
         # create the part usd object
-        partname = self.clean_name(name + part_hash.hexdigest())
+        partname = self.clean_name(name + part_hash.hexdigest()) + "_l"

```

Comment:
This was the critical issue.  The name of the object needed to be different from the triangles object to make sure this gets generated at all.